jupyter_book/yaml.py

```diff
@@ -63,6 +99,19 @@ def yaml_to_sphinx(yaml):
     # Update the theme options in the main config
     sphinx_config["html_theme_options"] = theme_options
 
+    # parse and rendering
+    parse = yaml.get("parse")
+    if parse:
+        if parse.get("myst_extended_syntax") is True:
```

Comment:
I wonder if we should have a single section that lists all of the things that are activated if `myst_extended_syntax` is true. We refer to it throughout the docs but in a piecemeal fashion.

Reply:
Well that single section is essentially: https://myst-parser.readthedocs.io/en/latest/using/syntax-optional.html. 
This is referred to in the "Other MyST Markdown syntax" heading in docs/content/myst.md.
Should I make it more explicit?

Reply:
let's just leave it for now and see if it creates confusion in folks over time, I think it's good 👍